sources/niUI/src/theoraplayer/TheoraVideoManager.h

```diff
@@ -15,7 +15,8 @@ the terms of the BSD license: http://opensource.org/licenses/BSD-3-Clause
 TheoraVideoClip* Theora_createVideoClip(ni::iFile* data_source,
                                         TheoraOutputMode output_mode,
                                         int numPrecachedOverride,
-                                        bool usePower2Stride);
+                                        bool usePower2Stride,
+                                        bool decodeFirstFirstFrame = true);
```

Comment:
No default param. Maybe those two flags would be better an enum. What does decodeFirstFrame do? - also it looks like theres a typo 'decodeFirstFirstFrame'.